internal/fields/fields.go

```diff
@@ -93,12 +93,8 @@ func findStruct(o types.Object) *types.Struct {
 	case *types.Struct:
 		return t
 	case *types.Slice:
-		s, ok := t.Elem().Underlying().(*types.Struct)
-		if !ok {
-			return nil
-		}
-		return s
```

Comment:
Was removing this block intentional? Could you explain why?

Reply:
Phew, glad I asked haha. I'm very rusty on this code so I almost assumed it was fine. 😆